spec/models/labware_creators/custom_tagged_plate_spec.rb

```diff
@@ -232,17 +233,18 @@ def expect_state_change_creation
           let(:tag_plate_state) { 'exhausted' }
 
           it 'creates a tagged plate' do
+            # This one will be VERY different
```

Comment:
Intentional comment?

Reply:
Well it came from removed line 241 below and I'm making the same expectation, just via API V2, so hopefully it's intentional.

Reply:
Yeah just not sure what the difference is that the comment is suggesting.

Reply:
I don't know.  I didn't want to try to clean up existing comments and I assumed it was made for a reason.  It's very easy to put in comments that aren't useful in the future 😄